packages/browser/src/modules/logto-client.ts

```diff
@@ -0,0 +1,28 @@
+import { OidcConfigResponse, Requester } from '@logto/js';
+
+export type LogtoConfig = {
+  endpoint: string;
+  clientId: string;
+  scopes?: string[];
+  resources?: string[];
+  usingPersistStorage?: boolean;
+  requester: Requester;
+};
+
+export type AccessToken = {
+  token: string;
+  scope: string;
+  expiresAt: number; // Timestamp in seconds
+};
+
+export default class LogtoClient {
+  protected accessTokenMap = new Map<string, AccessToken>();
+  protected refreshToken?: string;
+  protected idToken?: string;
+  protected logtoConfig: LogtoConfig;
+  protected oidcConfig?: OidcConfigResponse;
+
+  constructor(logtoConfig: LogtoConfig) {
+    this.logtoConfig = logtoConfig;
+  }
+}
```

Comment:
Wait, I thought this would be the `index.ts` of the browser package, but now I realized it's not. Why wouldn't you put this LogtoClient class in the index?

Reply:
u'r right. better with `index.ts` right now.